checkov/terraform/checks/resource/aws/IAMUserNotUsedForAccess.py

```diff
@@ -0,0 +1,30 @@
+from typing import List, Any
+
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+
+
+class IAMUserNotUsedForAccess(BaseResourceNegativeValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure IAM users are not defined " \
+               "AWS Access should be controlled by roles via an SSO, inline with AWS security best practices " \
+               "Require human users to use federation with an identity provider to access AWS using temporary credentials "
+        id = "CKV_AWS_272"
```

Comment:
```suggestion
        id = "CKV_AWS_273"
```
a check with that ID was merged today 🙂